src/Installers/Windows/SharedFramework/Product.wxs

```diff
@@ -73,6 +73,12 @@
 
             <?define ProductVersionKey=SOFTWARE\Microsoft\ASP.NET Core\Shared Framework\v$(var.MajorVersion).$(var.MinorVersion)\$(var.PackageVersion)?>
 
+            <?ifdef ProductVersionCentralKey?>
+            <?undef ProductVersionCentralKey?>
+            <?endif?>
+
+            <?define ProductVersionCentralKey=SOFTWARE\WOW6432Node\dotnet\Setup\InstalledVersions\$(var.Platform)\sharedfx?>
```

Comment:
Take out the WOW6432Node. You should let the component set the proper attribute instead of bypassing the WOW redirection. You can move this into the RegistryKey directly - it's unlikely for this value to change.